### DISCUSSION TITLE
Organization Members Search, Not Work with Large List (2024.5.0 Webvault - Latest Testing)

### DISCUSSION THREAD

**Discussion by gigaion** [dani-garcia/vaultwarden#4636] [Q&A] [answered]
Hello everyone,

I wanted to reach out to see if anyone on the testing docker image with the latest 2024.5.0 web vault has issues for anyone else?

It seems when you have in the range of 1500+ users (maybe affected with less), the members search feature does not work from the Organization admin console.

**Comment by stefan0xC** [comment, marked as answer]
I have not run into that issue myself (because I don't have any organization with that many users). But this is probably a bug in the web-vault which might have been fixed by https://github.com/bitwarden/clients/pull/9230 in `web-v2024.5.1` (cf. https://github.com/dani-garcia/bw_web_builds/pull/166).

**Reply by gigaion** [reply]
Thank you for providing insight into that. I will wait for v2024.5.1 to be merged into Vaultwarden if that is the case.
